### PR TITLE
Added cta and testimonials sections to the landing page

### PR DESCRIPTION
**Pull request by Lidiia92**

### CODE REVIEW

**Review by andrewlowell — APPROVED** [review]
Looks good :)

**Comment by andrewlowell** [comment]
Definitely need to pull from master before we merge.

**Review by dan2014 — CHANGES_REQUESTED** [review]
Logo_Syntax_Error
Can you remove the logo? It has a syntax error

File: src/Components/Views/LandingPage/LandingPageContent.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const LandingPageContent = props => {
             <div className="landingContent">
                 <div className="cta">
                     <header className="header">
-                        <img src={logo} />
+
```
Comment (dan2014) [inline src/Components/Views/LandingPage/LandingPageContent.js:23]: Can you add `ChallengeJS` as text and style it with`font-size: 2.5rem;`

**Review by dan2014 — CHANGES_REQUESTED** [review]

File: src/Components/Views/LandingPage/LandingPageContent.js (hunk 1 above)
Comment (dan2014) [reply src/Components/Views/LandingPage/LandingPageContent.js:23]: Don't worry about it. We'll change it ourselves.

**Review by dan2014 — APPROVED** [review]

**Review by Anubhav311 — APPROVED** [review]

**Pull request merged**